libeos-payg/tests/manager.c

```diff
@@ -274,6 +274,36 @@ get_code_format (Fixture *fixture)
   return g_steal_pointer (&regex);
 }
 
+/* test_code_format_prefix:
+ *
+ * Tests that the :code-format-prefix is properly returned.
+ */
+static void
+test_manager_code_format_prefix (Fixture *fixture,
+                         gconstpointer data)
+{
+  manager_new (fixture);
+
+  const gchar *code_format_prefix = epg_provider_get_code_format_prefix (fixture->provider);
+
+  g_assert_nonnull (code_format_prefix);
```

Comment:
You can make a stronger assertion here:

```suggestion
  g_assert_cmpstr (code_format_prefix, ==, "");
```